<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregarSitios</class>
 <widget class="QWidget" name="agregarSitios">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>689</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>836</width>
    <height>689</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Añadir Sitios</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>811</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>FORMULARIO DE SITIOS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="seleccionarImagen">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string> Seleccionar imagen...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../recursos/iconos/ic_imagen.png</normaloff>../recursos/iconos/ic_imagen.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="img_sitio">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(53, 53, 53);
border: 5px solid qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="desc_sitio">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>681</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 53, 53);
border:2px solid rgb(0, 85, 255);
border-radius: 5px</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guardarSitio">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>  Guardar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../recursos/iconos/ic_guardar.png</normaloff>../recursos/iconos/ic_guardar.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="descartar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string> Descartar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../recursos/iconos/ic_descartar.png</normaloff>../recursos/iconos/ic_descartar.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>DESCRIPCIÓN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="nombreLabel">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nombreLineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-bottom: 2px solid rgb(0, 170, 255);
border-left:none;
border-right:none;
border-top:none;
background-color: rgb(45, 45, 45);
padding-left:5px;
padding-right:5px;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ciudadLabel">
      <property name="text">
       <string>Ciudad</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="ciudadComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-bottom: 2px solid rgb(0, 170, 255);
border-left:none;
border-right:none;
border-top:none;
background-color: rgb(45, 45, 45);
padding-left:5px;
padding-right:5px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>descartar</sender>
   <signal>clicked()</signal>
   <receiver>agregarSitios</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
